apps/interactiveevents/templates/a4_candy_interactive_events/module_detail.html

```diff
@@ -14,14 +14,22 @@
         <div class="row justify-content-center">
             {% if extra_fields.event_image %}
                 <div class="col-md-5 my-2">
-                    <img src="/media/{{ extra_fields.event_image }}" />
+                    <img width="100%" src="{% thumbnail extra_fields.event_image 'item_image' %}" />
                 </div>
             {% endif %}
             <div class="col-md-5 my-2">
                 {% for phase in module.phases %}
                     {% if phase == module.active_phase and extra_fields.live_stream %}
                         <span class="label label--live">•{% trans ' Live now' %}</span>
                         &nbsp;
+                    {% elif not phase == module.active_phase %}
+                        <span class="label label--info">
+                            {% if not module.past_phases or module.future_phases %}
+                                In {{ week_countdown }} weeks
```

Comment:
We do have a property from a4, we use in the module tiles. I will find it!
Also, if you wanted to do it like that, you still need to make the string translatable. That's done with the blocktrans tag, where you can also add variables to the text.